sources/getRole.ts

```diff
@@ -14,67 +15,118 @@ export function getLocalName(element: Element): string {
 }
 
 const localNameToRoleMappings: Record<string, string | undefined> = {
+	address: "group",
 	article: "article",
+	// WARNING: Only in certain context
 	aside: "complementary",
+	b: "generic",
+	bdi: "generic",
+	bdo: "generic",
+	blockquote: "blockquote",
+	body: "generic",
 	button: "button",
+	caption: "caption",
+	code: "code",
+	data: "generic",
 	datalist: "listbox",
+	// WARNING: html-aria and html-aam conflict on this role assignment
```

Comment:
TODO: ref https://github.com/w3c/html-aam/pull/376 and https://github.com/w3c/html-aam/issues/373 (if still stand once using stable)